imblearn/under_sampling/_prototype_selection/_edited_nearest_neighbours.py

```diff
@@ -1,4 +1,4 @@
-"""Class to perform under-sampling based on the edited nearest neighbour
+"""Classes to perform under-sampling based on the edited nearest neighbour
```

Comment:
The docs and API use both the English and American form of the word neighbo(u)rs. Sklearn has american in its classes, imbalanced learn uses english in the classes names and american in the parameters. The docstrings had a bit of both.

I unified the docstrings with the English version Neighbours, at some point it would be good to consider unifying also the class name and params for one of the forms.